docs/src/components/DocLayout/index.tsx

```diff
@@ -90,157 +77,151 @@ export default function DocLayout({
         description={description}
         path={path}
       />
-      <ThemeProvider theme={theme}>
-        <DevModeProvider>
```

Comment:
There's a bunch of indentation changes in this file, but there's not a lot going on aside from removing these two context providers and updating the table of contents, so I recommend telling GitHub to hide whitespace changes while reviewing.